anti-analysis/anti-av/unhook-a-dll.yml

```diff
@@ -0,0 +1,26 @@
+rule:
+  meta:
+    name: unhook a DLL
+    namespace: anti-analysis/anti-av
+    authors:
+      - jakub.jozwiak@mandiant.com
+    scopes:
+      static: function
+      dynamic: thread
+    att&ck:
+      - Defense Evasion::Impair Defenses::Disable or Modify Tools [T1562.001]
+    mbc:
+      - Defense Evasion::Disable or Evade Security Tools [F0004]
+    references:
+      - https://www.ired.team/offensive-security/defense-evasion/how-to-unhook-a-dll-using-c++
+    examples:
+      - 282c32269a9893c5741ca682268369421c43ac21d73b1f6c23386d61c93bf3e9:0x1400014A4
+  features:
+    - and:
+      - api: GetModuleHandle
```

Comment:
add `GetModuleHandleEx`?

Reply:
Added in https://github.com/mandiant/capa-rules/pull/921/commits/3a03215dd576ad790256398d8b4999bffcb16e1d